images/nominatim/Dockerfile

```diff
@@ -133,6 +133,7 @@ COPY init.sh /app/init.sh
 COPY start.sh /app/start.sh
 COPY startapache.sh /app/startapache.sh
 COPY startpostgres.sh /app/startpostgres.sh
+COPY local.php /app/src/build/settings/local_db.php
```

Comment:
Why is this needed @Rub21?

Reply:
@geohacker  according to the doc, we ned the configuration  to connect to the DB,   when you run your nominating locally you can have the local.php file in your local directory  and make a copy like here 👉  `https://github.com/mediagis/nominatim-docker/blob/master/3.6/startapache.sh#L2`  but when you run  in other environments such us Kubernetes you won’t  have the local.php in the cluster..  the idea is to have this local.php in the container, and when i run the container it will take that config file, in our case as `export NOMINATIM_SETTINGS=‘/app/src/build/settings/local_api.php’`